code/__DEFINES/reagents.dm

```diff
@@ -47,8 +47,30 @@
 
 //used by chem masters and pill presses
 //update this if you add more patch icons
-#define PATCH_STYLE_LIST list("bandaid", "bandaid_brute", "bandaid_burn", "bandaid_both") //icon_state list
-#define DEFAULT_PATCH_STYLE "bandaid"
+#define PATCH_STYLE_LIST list(\
+	"bandaid_1", \
+	"bandaid_2", \
+	"bandaid_3", \
+	"bandaid_4", \
+	"bandaid_blank", \
+	"bandaid_both", \
+	"bandaid_brute", \
+	"bandaid_brute_2", \
+	"bandaid_burn", \
+	"bandaid_burn_2", \
+	"bandaid_clown", \
+	"bandaid_colonthree", \
```

Comment:
this is the worst thing i've ever seen

Reply:
I was told this is the proper way to make define lists

Reply:
Define lists are fundamentally not a good idea. You didn't make this code so you don't have to fix it but the usual proper way to make define lists is not at all

Reply:
i think he means the :3 sticker